manifests/bosh-manifest/deployments/aws/000-bosh-infrastructure.yml

```diff
@@ -3,8 +3,8 @@ resource_pools:
 - name: vms
   network: private
   stemcell:
-    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent?v=3197
-    sha1: 89a2210b8caf3884855d7db6d48b8863202c7783
+    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent?v=3215
+    sha1: 84c51fed6342d5eb7cd59728c7d691c75b6c1de8
```

Comment:
This bumps the stemcell used for bosh itself, but not for the CF deployment. We should bump both (and possibly also the stemcell used for the deployer-concourse)